crates/runtime/src/accelerated_table/refresh_task_runner.rs

```diff
@@ -118,6 +117,10 @@ impl RefreshTaskRunner {
                             let request = Self::create_refresh_from_overrides(Arc::clone(&base_refresh), overrides_opt).await;
                             task_completion = Some(Box::pin(refresh_task.run(request)));
                         }
+                        else => {
+                            // The parent refresher is shutting down, we should too
+                            break;
+                        }
```

Comment:
This is needed because when the parent Refresher exits after the initial load, the `on_start_refresh` channel isn't valid anymore and Tokio will panic if there isn't an `else` to handle that condition.